blocks/api/usage.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Internal dependencies
+ */
+import { getBlockType, getBlockTypes } from './registration';
+
+let recentBlockNames = [];
```

Comment:
Sounds like we should, this isn't internal block state or anything... Are we ok with doing this in the next iteration? The majority of the code in the usage module is going to be replaced with calls to a server side API so we can store a user's block usage (and do things like, initially populate the recently used blocks with their most frequently used blocks).